app/models/manageiq/providers/redhat/infra_manager/provision_workflow.rb

```diff
@@ -61,14 +61,20 @@ def allowed_customization_templates(options = {})
     end
   end
 
-  def allowed_hosts_obj(options = {})
-    return [] if (src = resources_for_ui).blank?
-    return [] unless src[:vm]
+  def allowed_datacenters(_options = {})
+    #byebug_term
```

Comment:
If it is never intended to be merged I would not make a PR to ManageIQ.  You can make a pull request to your own fork for those purposes.